File: giorno 2/g1/javascript/g1.js
const readline = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout
});

let a = 0, b = 0, c = 0;

readline.question('Metti 1 numero: ', (aInput) => {
  a = parseInt(aInput, 10);
  readline.question('Metti 1 numero: ', (bInput) => {
    b = parseInt(bInput, 10);
    readline.question('Metti 1 numero: ', (cInput) => {
      c = parseInt(cInput, 10);

      if (a > b && a > c) {
        console.log(a + ' è il numero più grande');
      }
      if (b > c && b > a) {
        console.log(b + ' è il numero più grande');
      }
      if (c > b && c > a) {
        console.log(c + ' è il numero più grande');
      }

      readline.close();
    });
  });
});

File: giorno 2/g1/python/g1.py
a = 0
b = 0
c = 0
a = int(input("Metti 1 numero: "))
b = int(input("Metti 1 numero: "))
c = int(input("Metti 1 numero: "))

if a > b and a > c:
    print(f"{a} è il numero più grande")
if b > c and b > a:
    print(f"{b} è il numero più grande")
if c > b and c > a:
    print(f"{c} è il numero più grande")